balena-docs/custom-build.md

```diff
@@ -1,3 +1,24 @@
 # Building Your Own Image
 
+To build your own custom balenaOS image, refer to the README for respective device type repository hosted in the [balenaOS GitHub org](https://github.com/balena-os/) for more information.
+
+Identifying the right repository is important. All device type repositories will start with `balena-*`. As we build device types of similar specification or board manufacturers together inside one repository. For example, to find the right repository for the device type Raspberry Pi 3, there will be a file in the repository named in the format `machine-name.coffee`. Here are the steps to finding the right repository for instructions:
```

Comment:
> As we build device types of similar specification or board manufacturers together inside one repository.

That phrase does not make complete sense. I would replace with:

> All device type repositories will start with `balena-*`, and all device types built by a repository define a device type coffee file named `machine-name.coffee` with device meta-data for the build pipeline and balenaCloud to injest. Find the correct coffee file for your machine name and read the `slug` field from it.
For example, to find the right repository for the device type Raspberry Pi 3,
...